app/utils/pages.tsx

```diff
@@ -71,3 +72,19 @@ export function getPages(): ILeagueLink[] {
     return paths;
 }
 
+export async function getUrlParams  (){
+    const leagueConfigurationKeys = await getLeagueConfigurationKeys();
```

Comment:
As we talked about, I think we should pull this out and that way we can make this function non-async. This will also make this function more testable and leave each template page to source the appropriate data to determine which pages should be generated.

Reply:
Fixed